src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
```

Comment:
To improve the code, you can add filtering of empty lines after splitting by semicolon.
```suggestion
  const declarations = sourceString
    .split(';')
    .filter(declaration => declaration.trim() !== '');
```